lottie/src/main/java/com/airbnb/lottie/LottieCompositionFactory.java

```diff
@@ -116,6 +119,21 @@ public static void setMaxCacheSize(int size) {
     }
   }
 
+
+
+  @WorkerThread
+  public static LottieResult<LottieComposition> fromAssetSyncMoshi(Context context, String fileName) {
```

Comment:
right now it is parallel to the old path, once I can harden it I will replace remove the legacy code